packages/puppeteer-core/src/node/ProductLauncher.ts

```diff
@@ -351,6 +351,9 @@ export abstract class ProductLauncher {
     const BiDi = await import(/* webpackIgnore: true */ '../bidi/bidi.js');
     const bidiConnection = await BiDi.connectBidiOverCdp(connection, {
       acceptInsecureCerts: opts.ignoreHTTPSErrors ?? false,
+      unhandledPromptBehavior: {
```

Comment:
We should also make the change in `Browser.ts` as we send request for capabilities there as that is where we request it for Firefox.

Reply:
firefox does not support object in `unhandledPromptBehavior` yet, so this would break ff alltogether.

Reply:
Is `unhandledPromptBehavior` old if it's new its should not fail due to Extendable CDDL?

Reply:
`unhandledPromptBehavior` used to be a string, but the recent change in BiDi made it possible to be a dict. We can pass the string fo ff, but I'm not sure if it makes sense